test/team_collection_test.rb

```diff
@@ -1,5 +1,57 @@
 require "./test/test_helper"
+require "mocha/minitest"
+require "./lib/team_collection"
 
 class TeamCollectionTest < Minitest::Test
-  
+
+  def test_it_exists
+    TeamCollection.stubs(:create_teams).returns(Array.new(32, mock('team')))
+    stubbed_team_collection = TeamCollection.new("./data/teams.csv")
+    assert_instance_of TeamCollection, stubbed_team_collection
+  end
+
+  def test_it_has_attributes
+    TeamCollection.stubs(:create_teams).returns(Array.new(32, mock('team')))
+    stubbed_team_collection = TeamCollection.new("./data/teams.csv")
+
+    assert_instance_of Array, stubbed_team_collection.teams
+    assert_equal 32, stubbed_team_collection.teams.length
+  end
+
+  def test_csv_loads
+    team_collection = TeamCollection.new("./data/teams.csv")
+
+    team_a = team_collection.teams.first
+    team_z = team_collection.teams.last
+
+
+    assert_equal 1,                       team_a.team_id
```

Comment:
is the spacing intentional on the assertions?

Reply:
It looks like it is an organizational approach but it looks a little strange to me